docs/excel/excel-add-ins-core-concepts.md

```diff
@@ -32,6 +32,16 @@ Excel.run(function (context) {
 });
 ```
 
+### Run options
+
+**Excel.run** has an overload that takes in a [RunOptions](https://docs.microsoft.com/javascript/api/excel/excel.runoptions) object. This contains a set of properties affecting platform behavior during the function's execution. The following are some of the most commonly used option properties:
```

Comment:
"The following are some of the most commonly used option properties:"  makes it sound like you're going to list multiple properties...but then there's only one property listed. Suggest you rephrase to something like -> "The following property is currently supported:"

Reply:
I debated about this for a while. I'm hesitant to say there's only one property in the event we add more and forget to update the article (or don't want to update the article because the new property covers some minor edge case).